maven-plugins/wsdl-validator-plugin/pom.xml

```diff
@@ -29,6 +29,9 @@
         <artifactId>cxf-maven-plugins</artifactId>
         <version>3.2.5-SNAPSHOT</version>
     </parent>
+    <properties>
+        <cxf.module.name>org.apache.cxf.wsdl.validator.plugin</cxf.module.name>
+    </properties>
```

Comment:
org.apache.cxf.plugin.wsdl-validator

Reply:
- is an issue in module names so looks good IMHO

Reply:
:+1: